File: src/main/java/no/novari/flyt/instance/gateway/model/vigo/Personalia.java
package no.novari.flyt.instance.gateway.model.vigo;

import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.extern.jackson.Jacksonized;

@Getter
@Jacksonized
@EqualsAndHashCode
@Builder
public class Personalia {

    private String fodselsnummer;
    private String fornavn;
    private String mellomnavn;
    private String etternavn;
    private String fodselsdato;
}

File: src/main/kotlin/no/fintlabs/instance/gateway/model/vigo/Personalia.kt
package no.fintlabs.instance.gateway.model.vigo

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Personalia(
    val fodselsnummer: String? = null,
    val fornavn: String? = null,
    val mellomnavn: String? = null,
    val etternavn: String? = null,
    val fodselsdato: String? = null
)
